packages/core/src/lib/utils/bech32.ts

```diff
@@ -0,0 +1,13 @@
+import bech32 from 'bech32';
+
+export const toBech32 = (address: Uint8Array): string => {
+	const words = bech32.toWords(address);
+	return bech32.encode('sei', words);
+};
+
+export const validateBech32 = (bech32Address: string) => {
```

Comment:
why is there two different methods to validate an address? `isValidAddress` in `address.ts` and `validateBech32` here